cluster/node-pools/master-default/stack.yaml

```diff
@@ -0,0 +1,70 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Kubernetes default master node pool
+
+Mappings:
+  Images:
+    eu-central-1:
+      LatestCoreOSImage: ami-604e118b
+      StableCoreOSImage: ami-862140e9
+
+Resources:
+  AutoScalingGroup:
+    CreationPolicy:
+      ResourceSignal:
+        Count: '0'
+        Timeout: PT15M
+    Properties:
+      HealthCheckGracePeriod: 480
+      HealthCheckType: EC2
+      LaunchConfigurationName: !Ref AutoScalingConfig
+      LoadBalancerNames:
+      - !ImportValue '{{ .Cluster.ID }}:master-load-balancer'
+      MinSize: '{{ .NodePool.MinSize }}'
+      MaxSize: '{{ .NodePool.MaxSize }}'
+      VPCZoneIdentifier: !Split [",", "{{ .Cluster.ConfigItems.subnets }}"]
+    Type: 'AWS::AutoScaling::AutoScalingGroup'
+  AutoScalingConfig:
+    Properties:
+      AssociatePublicIpAddress: true
+      BlockDeviceMappings:
+      - DeviceName: /dev/xvda
+        Ebs:
+          VolumeSize: 20
+          VolumeType: standard
+      EbsOptimized: false
+      IamInstanceProfile: !Ref AutoScalingInstanceProfile
+      # ImageId: " .AMI }}" # TODO: find
+      ImageId: "ami-862140e9"
```

Comment:
Should this use one of the mappings above?

Reply:
This is open because we don't have a clear process for how to define the AMI since the incident some weeks ago caused by a AMI unable to boot. For now I would just hardcode it here (and remove the mappings on top) and then we need to figure out how to go forward with this. This would be in line with the current approach were we hardcode it in a mapping because of senza (https://github.com/zalando-incubator/kubernetes-on-aws/pull/965).

Thoughts are welcome! (maybe this is not the right place to discuss it).

Reply:
Changed it to look this up from the mapping. Still we need to iterate on how to do this moving forward.